e2e-test/src/test/features/FileToDeduplicate.feature

```diff
@@ -0,0 +1,57 @@
+@Deduplicate
+Feature: Deduplicate - Verification of Deduplicate pipeline with File as source and BigQuery as sink
+
+  @FILE_SOURCE_TEST @BQ_SINK_TEST @PLUGIN-1207
+  Scenario: To verify complete flow of data extract and transfer from File source to BQ sink using Deduplicate Plugin
```

Comment:
we should focus more on testing deduplicate plugin functionality instead of testing various sources and sinks with the same dedup plugin config.

some things to test:
- different filter operations 
- specify numPartitions param. What does this do?

Reply:
Done.